### PR TITLE
[IMP] mail: pass MessageInReplyToViewComponent/localId

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2831082

### CODE REVIEW

File: addons/mail/static/src/components/message_in_reply_to_view/message_in_reply_to_view.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ export class MessageInReplyToView extends Component {
 }
 
 Object.assign(MessageInReplyToView, {
-    props: { messageInReplyToViewLocalId: String },
+    props: { localId: String },
     template: "mail.MessageInReplyToView",
```
Comment (phenix-factory) [inline addons/mail/static/src/components/message_in_reply_to_view/message_in_reply_to_view.js:19]: This is not enough, you need to update the `messageInReplyToView` getter too

**Comment by phenix-factory** [comment]
This show that why need to test the reply feature, because it's not working at all and it's not catched by runbot.

**Comment by alexkuhn** [comment]
> This show that why need to test the reply feature, because it's not working at all and it's not catched by runbot.

Just added a basic test with presence of `MessageInReplyToView`.

**Review by phenix-factory — APPROVED** [review]
LGTM

**Comment by alexkuhn** [comment]
Moved to grouped PR: https://github.com/odoo/odoo/pull/90320

**Pull request closed without merging**